### DISCUSSION TITLE
Penam-Monteith and mHM

### DISCUSSION THREAD

**Discussion by emresalkim** [mhm-ufz/mHM#65] [Q&A]
Dear all,

Please correct me if I am following a wrong path. As far as I understood while running mHM, in case I don't have PET data, mHM can calculate PET (Potential Evapotranspiration) for me. Also, it gives 3 options to do so. Let's say I have chosen Penam-Monteith (which is promising), I would like to discuss more on the needed variables.

First, net radiation:
I have surface solar radiation downwards and surface thermal radiation downwards (which I think is the greenhouse effect) -- I believe I have to sum these two together to calculate net radiation on surface. Am I right?

Second, absolute vapor pressure:
How may I supply mHM with this? Would multiplying the surface pressure with surface specific humidity (dimensionless) value help?

Third, for wind speed data, I believe the wind direction is not needed. Am I correct?

Thanks in advance,

Emre

**Comment by emresalkim** [comment]
Wanted to make a friendly reminder that my question still exists

**Comment by rohinikumar82** [comment]
Dear Emre,
Thank you for your remeinder message. Below you find the responses to your specific querries based on my readings (I have not used Penam-Monteith for a long time in mHM). 

First, net radiation:
I have surface solar radiation downwards and surface thermal radiation downwards (which I think is the greenhouse effect) -- I believe I have to sum these two together to calculate net radiation on surface. Am I right?

Resposne: No. Some adjustements are needed. To get the net radiation, you need to estimate outgoing solar and thermal radiation term as well; and then you need to sum of all incoming and outgoing radiation fluxes at the land surface (ofcourse take care of the sign). Check for e.g., https://open.library.okstate.edu/rainorshine/chapter/11-3-net-radiation/ 

Second, absolute vapor pressure:
How may I supply mHM with this? Would multiplying the surface pressure with surface specific humidity (dimensionless) value help?

Response: You need to take care of specific details (constants, formulations, etc)  in this converion process. Check details (e.g., here): https://cran.r-project.org/web/packages/humidity/vignettes/humidity-measures.html

Third, for wind speed data, I believe the wind direction is not needed. Am I correct?
Response: Possibly Yes. Please check the code. 

Hope this helps, solving your issues. 

thanks.

Best,
Rohini

**Reply by emresalkim** [reply]
Dear Dr Kumar,

Thank you very much for your reply.
I will investigate your answers soon.

Yours,

Emre

**Reply by pallavhydro** [reply]
Hi @emresalkim 

If the discussion here solved your issue, I'd appreciate you mark the response that solved the issue.

Thanks in advance.
Pallav
